Library/Formula/freealut.rb

```diff
@@ -36,3 +36,4 @@ def install
  
  ################################################################################
  # Checks for header files.
+ 
```

Comment:
Careful with the whitespacing here. Inline patches are very delicate about accidental whitespacing.